tests/fiveshell/shell_example.py

```diff
@@ -1,15 +1,19 @@
-# %% Problem Set-up
+# Problem Set-up
 import numpy as np
 import matplotlib.pyplot as plt
 import seaborn as sns
 import pandas as pd
+import os
```

Comment:
order of imports should always be:
python modules
modules installed via pip
modules from this package